### PR TITLE
Fix/design

### PR DESCRIPTION
**Pull request by nikita1tup**

### CODE REVIEW

File: src/views/DataSources.vue [hunk 1]
```diff
@@ -127,13 +127,13 @@ export default defineComponent({
...
     const [isLoading, lockLoading, releaseLoading] = useBooleanSemaphore()
-    const ITEMS_PER_PAGE = 4
+    const ITEMS_PER_PAGE = 50
     const currentPage = ref(1)
     const totalPages = ref(0)
     const dataSourcesCount = ref(0)
     const dataSources = ref([])
     const accountAddress = wallet.account.address
-    const sortingActivitiesValue = ref('')
+    const sortingActivitiesValue = ref('latest_updated')
```
Comment (ArtSuslov) [inline src/views/DataSources.vue:136]: ACTIVITIES_SORT.latest (and fix that enum, key should be presented in camel case)

File: src/views/OracleScripts.vue [hunk 2]
```diff
@@ -144,15 +144,15 @@ export default defineComponent({
...
+    const ITEMS_PER_PAGE = 50
     const currentPage = ref(1)
     const totalPages = ref()
     const oracleScriptsCount = ref(0)
     const oracleScripts = ref()
     const accountAddress = wallet.account.address
     const mostRequestedOracleScripts = ref()
     const oracleScriptsName = ref('')
-    const sortingActivitiesValue = ref('')
+    const sortingActivitiesValue = ref('latest_updated')
```
Comment (ArtSuslov) [inline src/views/OracleScripts.vue:155]: !

File: src/views/Proposal.vue [hunk 3]
```diff
@@ -25,11 +25,11 @@
           <div class="info-table__row-value info-table__row-value_flex">
             <a
               class="info-table__row-link"
-              :href="`${API_CONFIG.odinScan}/account/${proposal.proposerAddress}`"
+              :href="`${API_CONFIG.odinScan}/account/${proposal?.proposerAddress}`"
```
Comment (ArtSuslov) [inline src/views/Proposal.vue:28]: is it expected flow when there is no proposerAddress and link gets broken?

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

File: src/components/modals/ModalBase.vue [hunk 4]
```diff
@@ -2,7 +2,10 @@
   <div class="modal-base">
     <div class="modal-base__backdrop" @click="emitClose()"></div>
     <div class="modal-base__content">
-      <div class="modal-base__content-head fx-row mg-b32">
+      <div
+        class="modal-base__content-head fx-row"
+        :class="{ 'mg-b32': isAddMargin }"
```
[the remark this answers is not in the document]
Comment (nikita1tup) [reply src/components/modals/ModalBase.vue:7]: i need to change UndelegateFromModal, in this component i can't remove margin due to scope. If I remove the scope, then the margin will also be removed in other modals

File: src/components/LinksDropdown.vue [hunk 5]
```diff
@@ -49,80 +53,74 @@ export default defineComponent({
...
-    color: inherit;
-    font-weight: 400;
-    line-height: 2.4rem;
-    font-size: 1.6rem;
-    cursor: pointer;
+.arrow-icon {
+  fill: #212529;
+  transform: translate(0.3rem, -0.1rem) rotate(270deg);
+}
+.arrow-icon_active {
```
Comment (ArtSuslov) [inline src/components/LinksDropdown.vue:60]: discussed

File: src/components/LinksDropdown.vue [hunk 6]
```diff
@@ -49,80 +53,74 @@ export default defineComponent({
...
+  fill: var(--clr__action);
+}
+.nav__dropdown {
```
Comment (ArtSuslov) [inline src/components/LinksDropdown.vue:63]: lets make it the way - if you work on some component and classnames did not follow BEM, you just rename them all over that component

File: src/components/LinksDropdown.vue [hunk 7]
```diff
@@ -165,53 +163,53 @@ export default defineComponent({
...
-      padding: 2.4rem 1.2rem 2.4rem 4.8rem;
-      border-bottom: 0.1rem solid #ced4da;
-      &:hover {
-        background: rgba(204, 228, 255, 0.4);
-      }
+  }
+  .nav__dropdown-link {
+    width: 100%;
+    padding: 2.4rem 1.2rem 2.4rem 4.8rem;
+    border-bottom: 0.1rem solid #ced4da;
```
Comment (ArtSuslov) [inline src/components/LinksDropdown.vue:189]: var(--clr__input-border);

File: src/components/LinksDropdown.vue [hunk 8]
```diff
@@ -165,53 +163,53 @@ export default defineComponent({
...
-      width: 100%;
-      padding: 2.4rem 1.2rem;
-      border-bottom: 0.1rem solid #ced4da;
+  }
+  .nav__dropdown-wrapper {
+    text-align: center;
+    justify-content: space-between;
+    width: 100%;
+    padding: 2.4rem 1.2rem;
+    border-bottom: 0.1rem solid #ced4da;
```
Comment (ArtSuslov) [inline src/components/LinksDropdown.vue:199]: !

File: src/components/LinksDropdown.vue [hunk 9]
```diff
@@ -165,53 +163,53 @@ export default defineComponent({
...
+  }
+  .nav__link {
+    width: 100%;
+    padding: 2.4rem 1.2rem;
+    border-bottom: 0.1rem solid #ced4da;
```
Comment (ArtSuslov) [inline src/components/LinksDropdown.vue:204]: !

File: src/components/modals/UndelegateFormModal.vue [hunk 10]
```diff
@@ -160,4 +193,71 @@ export function showUndelegateFormDialog(
 }
 </script>
 
-<style scoped lang="scss"></style>
+<style scoped lang="scss">
+.copy-text {
```
[the remark this answers is not in the document]
Comment (ArtSuslov) [reply src/components/modals/UndelegateFormModal.vue:209]: ?

File: src/views/Wallet.vue [hunk 11]
```diff
@@ -10,7 +10,16 @@
         <div class="info__card-balance mg-b40">
           <div class="info__card-row">
             <span class="info__card-row-title">ODIN</span>
-            <span class="info__card-row-value">
+            <span
+              class="info__card-row-value"
```
Comment (ArtSuslov) [inline src/views/Wallet.vue:14]: component's name is Wallet, not Info

File: src/views/Wallet.vue [hunk 12]
```diff
@@ -34,10 +43,13 @@
     </div>
 
     <div class="view-main__subtitle mg-b24">
+      <div class="view-main__info">
```
Comment (ArtSuslov) [inline src/views/Wallet.vue:46]: same here

File: src/views/Wallet.vue [hunk 13]
```diff
@@ -360,23 +357,28 @@ export default defineComponent({
 
 .view-main {
   &__subtitle {
-    position: relative;
+    display: flex;
+    align-items: center;
   }
-
-  &__subtitle-item {
+  &__info {
```
Comment (ArtSuslov) [inline src/views/Wallet.vue:363]: no waterfalls

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

**Pull request merged**